vmc/constants/constants.go

```diff
@@ -10,9 +10,12 @@ const (
 	// DefaultCspURL defines the default URL for CSP.
 	DefaultCspURL string = "https://console.cloud.vmware.com"
 
-	// CspRefreshURLSuffix defines the CSP Refresh API endpoint.
+	// CspRefreshURLSuffix defines the CSP Refresh Token API endpoint.
 	CspRefreshURLSuffix string = "/csp/gateway/am/api/auth/api-tokens/authorize"
 
+	// CspOauthURLSuffix defines the CSP Oauth API endpoint.
+	CspOauthURLSuffix string = "/csp/gateway/am/api/auth/token"
```

Comment:
It's called the "token endpoint", so you should call it `CspTokenURL` or something like that (even though "CSP" is not a user facing term, "VMware cloud services" is)

Reply:
Addressed.